docs/glossary.md

```diff
@@ -18,6 +18,15 @@ easier to use in applications.
 Brightray is a low-level dependency of Electron that does not concern the
 majority of Electron users.
 
+### CRT
+
+It's a Microsoft Visual Studio thing.
```

Comment:
Ehh, to be precise, it's not "a Microsoft thing," as the citation from MSDN actually indicates. :)
You could call the **VC++ Runtime** a MS thing, and it _implements_ the CRT, which is defined by the standard.